plugins/module_utils/crypto/cryptography_openssh.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright: (c) 2021, Andrew Pantuso (@ajpantuso) <ajpantuso@gmail.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from base64 import b64encode, b64decode
+from getpass import getuser
+from socket import gethostname
+
+try:
+    from .basic import (
+        HAS_CRYPTOGRAPHY,
+        CRYPTOGRAPHY_HAS_ED25519,
+    )
+except ImportError:
+    HAS_CRYPTOGRAPHY = False
+    CRYPTOGRAPHY_HAS_ED25519 = False
+
+
+if HAS_CRYPTOGRAPHY and CRYPTOGRAPHY_HAS_ED25519:
+
+    from cryptography.exceptions import UnsupportedAlgorithm
+    from cryptography.hazmat.backends.openssl import backend
+    from cryptography.hazmat.primitives import hashes, serialization
+    from cryptography.hazmat.primitives.asymmetric import dsa, ec, rsa, padding
+    from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PrivateKey
+
+    _TEXT_ENCODING = 'UTF-8'
+
+    class InvalidAlgorithmError(Exception):
+        pass
+
+    class InvalidCommentError(Exception):
+        pass
+
+    class InvalidDataError(Exception):
+        pass
+
+    class InvalidKeyFileError(Exception):
+        pass
+
+    class InvalidKeySizeError(Exception):
+        pass
+
+    class InvalidKeyTypeError(Exception):
+        pass
+
+    class InvalidPassphraseError(Exception):
+        pass
+
+    class Asymmetric_Keypair(object):
+        """Container for newly generated asymmetric keypairs or those loaded from existing files
+
+           :path: A path to an existing private key to be loaded
```

Comment:
These are not properties of the class.

If you want to document the constructor arguments, this must be in the docstring for the constructor.